nonUniqueElements.js

```diff
@@ -25,6 +25,17 @@ nonUniqueElements([10, 9, 10, 10, 9, 8]) == [10, 9, 10, 10, 9]
  */
 
 export default function nonUniqueElements(data) {
-  // your solution goes here
-  return data
+  	var list = [],
+  	var	output = [];
```

Comment:
Так нельзя
```suggestion
  	var	output = [];
  	output = []
```

либо 
```js
var list = []  // без запятой
var output = []
```

Reply:
Опечатка, исправил
```suggestion
  	var list = []
  	var output = []
```